deeplabcut/pose_estimation_pytorch/runners/snapshots.py

```diff
@@ -113,41 +113,43 @@ def update(self, epoch: int, state_dict: dict, last: bool = False) -> None:
         ):
             current_best = self.best()
             self._best_metric = metrics[self._key]
+            
+            # Save the new best model
             save_path = self.snapshot_path(epoch, best=True)
             parsed_state_dict = {
-                k: v
+                k: v 
```

Comment:
You should run [black](https://github.com/psf/black) on the changes you made (with `black deeplabcut/pose_estimation_pytorch/runners/snapshots.py`) to lint the code (so there aren't these extra spaces)

Reply:
Thanks! Next time I will run black before I submit the changes.

<img width="699" alt="image" src="https://github.com/user-attachments/assets/98ccd398-1ef3-40a9-a93e-62b3b7cf37c0">